cdc/processor/pipeline/cyclic_mark.go

```diff
@@ -56,12 +57,13 @@ func (n *cyclicMarkNode) Init(ctx pipeline.NodeContext) error {
 	return nil
 }
 
-// Receive receives the message from the previous node
+// Receive receives the message from the previous node.
 // In the previous nodes(puller node and sorter node),
-// the change logs of mark table and normal table are listen by one puller,
-// and sorted by one sorter. So, this node will receive a commitTs-ordered stream which include the mark rows and normal rows.
-// Under the above conditions, we need to cache at most one transaction's rows to matching rows.
-// For every row event, Receive function flushes every the last transaction's rows, and adds the mark row or normal row into the cache.
+// the change logs of mark table and normal table are listen by one puller, and sorted by one sorter.
+// So, this node will receive a commitTs-ordered stream which include the mark row events and normal row events.
```

Comment:
Could you wrap line length within 80 chars?

Reply:
Maybe [lll](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/lll.go) will help. Let me fix it manually first.

Reply:
Done.